zcash_client_sqlite/src/lib.rs

```diff
@@ -835,6 +847,18 @@ impl<'conn, P: consensus::Parameters> WalletCommitmentTrees for WalletDb<SqlTran
             roots,
         )
     }
+
+    fn get_checkpoint_depth(
+        &self,
+        min_confirmations: NonZeroU32,
+    ) -> Result<usize, ShardTreeError<Self::Error>> {
+        get_checkpoint_depth(&self.conn.0, min_confirmations)
+            .map_err(|e| ShardTreeError::Storage(commitment_tree::Error::Query(e)))?
+            // `CheckpointPruned` is perhaps a little misleading; in this case it's that
+            // the chain tip is unknown, but if that were the case we should never have been
+            // calling this anyway.
```

Comment:
Does this comment apply to the other `WalletCommitmentTrees` impl as well?

Reply:
Ah, yeah, it's the same.